pandas/tests/tseries/test_offsets.py

```diff
@@ -2192,7 +2192,7 @@ def test_repr(self):
     def test_corner(self):
         pytest.raises(ValueError, Week, weekday=7)
         tm.assert_raises_regex(
-            ValueError, "Day must be", Week, weekday=-1)
+            ValueError, "weekday must be", Week, weekday=-1)
```

Comment:
Why are we changing all of these "Day" strings to "weekday" ?

Reply:
Because the incorrect argument is "weekday".  Many of these also take a "day" argument, so there is potential for ambiguity.